src/modules/accounting/accounting.py

```diff
@@ -366,13 +379,13 @@ def _calculate_report_v1(self, blockstamp: ReferenceBlockStamp) -> ReportData:
         extra_data_part_v1 = self._calculate_extra_data_report_v1(blockstamp)
         return self._combine_report_parts(1, blockstamp, rebase_part, modules_part, wq_part, extra_data_part_v1)
 
-    def _calculate_report_v2(self, blockstamp: ReferenceBlockStamp) -> ReportData:
+    def _calculate_report_latest_version(self, consensus_version: int, blockstamp: ReferenceBlockStamp) -> ReportData:
```

Comment:
consider using exact version number instead of `latest_version` and do explicit checks for the consensus version to be sure that non updated oracles will not report after next consensus version bump